src/util.cpp

```diff
@@ -2263,7 +2263,7 @@ QCString argListToString(const ArgumentList &al,bool useCanonicalType,bool showD
   if (al.volatileSpecifier()) result+=" volatile";
   if (al.refQualifier()==RefQualifierLValue) result+=" &";
   else if (al.refQualifier()==RefQualifierRValue) result+=" &&";
-  if (!al.trailingReturnType().isEmpty()) result+=al.trailingReturnType();
+  if (!al.trailingReturnType().isEmpty()) result+=" -> "+al.trailingReturnType();
```

Comment:
I don't think this fix is not a fix but an undo of #8013 and should not be present (It certainly has nothing tho do with hyphenation).

Reply:
Hmm, I don't have good knowledge about `doxygen` internals, but latest master branch may produce this kind of LaTeX:

![image](https://user-images.githubusercontent.com/871385/93015421-ff5d0180-f5c1-11ea-8dcd-5732bc817795.png)

Should I try proposed change in `src/utils.cpp` at line 2266 to avoid this formatting?

Reply:
Line 2266 of `util.cpp` has nothing to do with hyphenation, so your real proposed patch (at line 6300) should stay. I think you should update your pull proposed request so that the line 2266 of `src/utils.cpp` is not changed.

Reply:
Fine, I had no intention to change that line. Fixed.